src/pages/Homepage.tsx

```diff
@@ -2,7 +2,25 @@ import styled from "styled-components";
 import ContainerPrototype from "../components/prototypes/ContainerPrototype";
 import Header from "../components/homepage/header/Header";
 import PokemonPictureCard from "../components/homepage/pokemonPictureCards/PokemonPictureCard";
-import React from "react";
+import React, { useState } from "react";
+import pickRandomPokemonNumbers from "../functions/utilities/pickRandomPokemonNumbers";
+
+export default function Homepage(): React.ReactElement {
+    const [randomPokemonSelection, setRandomPokemonSelection] = useState<number[]>(pickRandomPokemonNumbers());
+
+    const mapPictureCards = (): React.ReactElement[] =>
+        randomPokemonSelection.map((i) => <PokemonPictureCard id={i} />);
```

Comment:
Again, no unique Key, please see console logs . There are errors related to this.